cumulusci/cli/cli.py

```diff
@@ -106,6 +106,23 @@ def check_latest_version():
             click.echo(
                 "An update to CumulusCI is available. Use pip install --upgrade cumulusci to update.")
 
+def check_org_expired(config, org_name, org_config):
+    if org_config.scratch and org_config.date_created and org_config.expired:
+        click.echo(click.style('The scratch org is expired', fg='yellow'))
+        if click.confirm('Attempt to recreate the scratch org?', default=True):
+            config.keychain.create_scratch_org(
+                org_name,
+                org_config.config_name,
+                org_config.days,
+            )
+            click.echo('Org config was refreshed, attempting to recreate scratch org')
+            org = config.keychain.get_org(org_name)
+            org.create_org()
+        else:
+            raise click.ClickException('The target scratch org is expired.  You can use cci org remove {} to remove the org and then recreate the config manually'.format(org_name))
+
+    return org_config
```

Comment:
is there a reason this is called in an `org_config = check_org_expired(config, org_name, org_config)` kinda way?

if it's not intended to mutate the state of the org_config var, lets not give it that power. can we just not return a value?

```
org_config = config.project_config.get_org(org_name)
check_org_expired(config, org_name, org_config)
org_config.refresh_oauth_token(config.keychain.get_connected_app())
```
instead of 
```
org_config = config.project_config.get_org(org_name)
org_config = check_org_expired(config, org_name, org_config)
org_config.refresh_oauth_token(config.keychain.get_connected_app())
```

Reply:
There was a bug in that check_org_expired wasn't returning the newly built org_config.  We want it return the new org_config so the click command function doesn't need to query the keychain again for the org.  I'll push a fix shortly.